res/app/control-panes/dashboard/shell/shell.pug

```diff
@@ -53,6 +53,21 @@
     uib-tooltip='{{ "window_animation_scale 1, transition_animation_scale 1, animator_duration_scale 1" | translate }}')
       i.fa.fa-play
       span(translate) Enable Animations
+
+    .clearfix
+    
+    button.btn.btn-xs.btn-info-outline(ng-click='run("wm density reset")',
+    uib-tooltip='{{ "wm density reset" | translate }}')
+      i.fa.fa-tablet
+      span(translate) Reset display density
+    button.btn.btn-xs.btn-info-outline(ng-click='run("settings put system font_scale 1")',
+    uib-tooltip='{{ "settings put system font_scale 1" | translate }}')
+      i.fa.fa-font
+      span(translate) Reset font size
+    button.btn.btn-xs.btn-info-outline(ng-click='run("am broadcast -a io.appium.settings.locale --es lang en --es country US")',
+    uib-tooltip='{{ "am broadcast -a io.appium.settings.locale --es lang en --es country US" | translate }}')
+      i.fa.fa-language
+      span(translate) Reset locale (requires Appium Settings installed)
```

Comment:
I would rather put this requirement into the tooltip so the name is not so long. I think the command can be skipped in tooltip as it doesn't add much value in comparison with the requirement for Appium Settings.

Also I would mention that it resets to en/US locale
```suggestion
    uib-tooltip='{{ "to en/US. Requires Appium Settings installed" | translate }}')
      i.fa.fa-language
      span(translate) Reset locale
```